wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/TeamImageView.swift

```diff
@@ -24,7 +24,7 @@ final class TeamImageView: UIImageView {
         case big
     }
 
-    // TODO [WPB-765]: Consider unifying this type with `AvatarImageView.Avatar`
+    // Maybe this type could eventually be merged with `AvatarImageView.Avatar`
```

Comment:
Without the "TODO" and ticket number, it feels like a statement rather than an open topic that needs resolving.

Reply:
Created a ticket and mentioned it here.